readme.md

```diff
@@ -0,0 +1,28 @@
+### Hetmec Front-end
+
+This repository contains the front-end interface for the [Hetmech project](https://github.com/greenelab/hetmech).
+
+
+#### Test/Build
+
+To install environment:
+
+1. Install latest version of the [Node.js platform](https://nodejs.org/en/)
+2. Clone this repo
+3. Navigate to your local clone
+4. Run `npm install` to download and install all the node package dependencies
+
+To build packaged/bundled app:
+
+1. Run `npx webpack`
+2. Wait for build to complete, then open `/build/index.html`.
```

Comment:
`./build/...` will be better, because the leading `/` indicates the absolute path.